src/main/java/domain/Cars.java

```diff
@@ -0,0 +1,49 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+
+    private List<Car> cars;
```

Comment:
가능한 곳은 최대한 불변을 유지해주세요.
```suggestion
    private final List<Car> cars;
```
불변을 유지하려는 이유는 무엇일까요?

Reply:
불변을 유지해야 하는 이유는 해당 변수가 잘못된 값이나 객체로 재할당되지 않게 하기 위함이라고 생각합니다!
의식적으로 늘 체크하고 불변을 유지해야 하는 변수는 잘 설정해야겠어요